build-test.cmd

```diff
@@ -104,7 +104,10 @@ if [!processedArgs!]==[] (
 :ArgsDone
 
 @REM Special handling for -priority=N argument.
-if %__Priority% GTR 0 (set "__PriorityArg=-priority=%__Priority%")
+if %__Priority% GTR 0 (
+    set "__PriorityArg=-priority=%__Priority%"
+    set "__PriorityMsbuildArg=/p:CLRTestPriorityToBuild=%__Priority%"
```

Comment:
The core issue was for MSBuild, the correct property is `CLRTestPriorityToBuild`, not `Priority`. The `priority` flag that was previously passed is mapped in the `config.json` file and set in the depths of `run.exe` 😖 unclear why this mapping logic exists.